tests/Task.test.ts

```diff
@@ -955,12 +955,12 @@ describe('check removal of the global filter', () => {
                 '- [ ] task with an extension of the global filter #task/with/extension',
             // This is not the behavior we eventually want, but this is the existing situation
             expectedDescription:
-                'task with an extension of the global filter /with/extension',
+                'task with an extension of the global filter #task/with/extension',
```

Comment:
I think the comment on line 956 should be deleted now. I haven't checked if there were any other occurrences of the same comment.

Reply:
Oops, done. (There aren't any more.)